.travis.yml

```diff
@@ -12,23 +12,30 @@ install:
 jobs:
   include:
   - stage: Tests
-    name: Validate
+    name: Unit tests
     script:
-    - make lint
+    - make test
 
-  - name: ES Smoke Tests
+  - name: Backend E2E Tests - Minikube
+    if: false
     script:
-    - scripts/setup-minikube-for-linux.sh
-    - make test CHARTS=enterprise-suite
+    - scripts/setup-minikube-for-linux.sh || travis_terminate 1
+    - make -C enterprise-suite minikube-backend-tests
 
-  - name: ES Smoke Tests (Latest)
+  - name: Backend E2E Tests - Openshift
+    script:
+      - scripts/setup-openshift.sh || travis_terminate 1
+      - make -C enterprise-suite openshift-backend-tests
+
+  - name: Backend E2E Tests - Minikube Latest
     # Only build latest in the daily cron - don't block PRs or master releases.
     if: type in (cron)
     script:
-    - scripts/setup-minikube-for-linux.sh
-    - make test CHARTS=enterprise-suite-latest
+    - scripts/setup-minikube-for-linux.sh || travis_terminate 1
+    - make -C enterprise-suite-latest minikube-backend-tests
 
-  - name: ES Frontend E2E
+  - name: Frontend E2E Tests - Minikube
+    if: false
```

Comment:
This here for testing?

Reply:
Yep, need to remove these when it's ready.